### DISCUSSION TITLE
Date selector cannot select dates in 2023

### DISCUSSION THREAD

**Discussion by devinhedge** [home-assistant/frontend#14573] [Dashboard] [answered]
SITUATION:
- When I go to select a date in 2023 for a date helper, I cannot.

STEPS TO RECREATE:
- Via iPhone Home Assistant app
- Create a date helper
- Add new date helper entity to Lovelace dashboard by adding the entity via the “Add Entity to Dashboard” using the default suggested cards.
- Navigate to the date entity in the dashboard 
- Click the Date Picker widget
- Attempt to select a date in 2023 (you cannot)

VERSION:
- iPhone Home Assistant App 2022.10.1
- Home Assistant 2022.11.5
- Supervisor 2022.11.2
- Operating System 9.3
- Frontend 20221108.0 - latest

**Comment by frenck** [comment, marked as answer]
I think you meant to report an issue... however, you started a discussion. If you did want to report an issue, please go to the issue tracker and open one there instead:

<https://github.com/home-assistant/frontend/issues>

../Frenck

**Reply by devinhedge** [reply]
May be related to #12329

**Reply by devinhedge** [reply]
Thanks. Silly app. Yes… that was the case.
